### PR TITLE
add replacement count matrix

### PR DESCRIPTION
**Pull request by ambrosejcarr**
* Replace dropseqtools with SC Tools

### CODE REVIEW

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
jenkins retest

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

**Comment by ambrosejcarr** [comment]
retest this please

File: library/tasks/CreateCountMatrix.wdl [hunk 1]
```diff
@@ -51,3 +50,95 @@ task DropSeqToolsDigitalExpression {
...
+    memory: "${machine_mem_mb} MB"
+    disks: "local-disk ${disk} HDD"
+    cpu: cpu
+    preemptible: preemptible
+  }
+
+  output {
+    File sparse_count_matrix = "sparse_counts.npz"
+  }
+}
```
Comment (rexwangcc) [inline library/tasks/CreateCountMatrix.wdl]: nit: new line at the end

**Pull request merged**